src/sass/_card.scss

```diff
@@ -10,11 +10,8 @@
 
   .card-image {
     background-repeat: no-repeat;
```

Comment:
Can the new changes be reverted?

Reply:
The new changes in the _card.scss file? I referenced the master branch to revert back to the file changes that was previously made for the card-images to be perfectly placed?

<img width="1106" alt="screen shot 2018-11-19 at 8 14 55 pm" src="https://user-images.githubusercontent.com/8063614/48745078-f4d8b780-ec37-11e8-9309-04db2c783c12.png">

Here's what the current updates look like locally on my end.